linode/manager
# fix: [M3-6480] - Fix blank Node Pool plan selection

## Description 📝

Fixes a bug introduced in #8840 that causes the `AddNodePool` drawer to not show any plans in certain situations.

Also included are basic unit tests that validate the fix and small amendments to the recent `useQueryClient` PR.

## Preview 📷

| Before      | After |
| ----------- | ----------- |
|![Screenshot 2023-04-17 at 1 30 24 PM](https://user-images.githubusercontent.com/122488130/232564232-18e6a2a3-d6bd-465d-bb98-297c8e8a5cc0.jpg)|![Screenshot 2023-04-17 at 1 30 07 PM](https://user-images.githubusercontent.com/122488130/232564301-475d8176-4cf7-45d0-8180-7f2d7acb4e3c.jpg)|

## How to test 🧪

1. Under Kubernetes, navigate to an existing cluster or create a new one.
2. Refresh the page to clean the RQ store.
3. Click on 'Add A Node Pool'.
4. Verify the `AddNodePoolDrawer` appears and displays all plan types and details.
